src/data/entity_registry.ts

```diff
@@ -24,6 +24,7 @@ export interface EntityRegistryDisplayEntry {
   translation_key?: string;
   platform?: string;
   display_precision?: number;
+  has_entity_name?: boolean;
```

Comment:
This doesn't seem used now right?

Reply:
I was used in entity picker but I removed it from this PR. I will be useful if we want to compute something similar to friendly_name in the front-end directly.